source/codegen/metadata/niscope/enums.py

```diff
@@ -18,276 +19,163 @@
-                'documentation': {
-                    'description': 'Butterworth IIR Filter'
-                },
                 'name': 'NISCOPE_VAL_BUTTERWORTH_FILTER',
                 'value': 4017
             },
             {
-                'documentation': {
-                    'description': 'Chebyshev IIR Filter'
-                },
                 'name': 'NISCOPE_VAL_CHEBYSHEV_FILTER',
                 'value': 4018
             },
             {
-                'documentation': {
-                    'description': 'FFT Amp. Spectrum (dB)'
-                },
                 'name': 'NISCOPE_VAL_FFT_AMP_SPECTRUM_DB',
                 'value': 4019
             },
             {
-                'documentation': {
-                    'description': 'Hamming Window'
-                },
                 'name': 'NISCOPE_VAL_HAMMING_WINDOW',
                 'value': 4020
             },
             {
-                'documentation': {
-                    'description': 'FIR Windowed Filter'
-                },
                 'name': 'NISCOPE_VAL_WINDOWED_FIR_FILTER',
                 'value': 4021
             },
             {
-                'documentation': {
-                    'description': 'Bessel IIR Filter'
-                },
                 'name': 'NISCOPE_VAL_BESSEL_FILTER',
                 'value': 4022
             },
             {
-                'documentation': {
-                    'description': 'Triangle Window'
-                },
                 'name': 'NISCOPE_VAL_TRIANGLE_WINDOW',
                 'value': 4023
             },
             {
-                'documentation': {
-                    'description': 'Blackman Window'
-                },
                 'name': 'NISCOPE_VAL_BLACKMAN_WINDOW',
                 'value': 4024
             },
             {
-                'documentation': {
-                    'description': 'Array Offset'
-                },
                 'name': 'NISCOPE_VAL_ARRAY_OFFSET',
                 'value': 4025
             },
             {
-                'documentation': {
-                    'description': 'Array Gain'
-                },
                 'name': 'NISCOPE_VAL_ARRAY_GAIN',
                 'value': 4026
             }
         ]
     },
     'CableSenseMode': {
+        'codegen_method': 'public',
         'values': [
             {
-                'documentation': {
-                    'description': 'The oscilloscope is not configured to emit a CableSense signal.'
-                },
                 'name': 'NISCOPE_VAL_CABLE_SENSE_MODE_DISABLED',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'The oscilloscope is configured to emit a single CableSense pulse.'
-                },
                 'name': 'NISCOPE_VAL_CABLE_SENSE_MODE_ON_DEMAND',
                 'value': 1
             }
         ]
     },
-    'CalibrationTypes': {
```

Comment:
Missing

Reply:
These enums are only used by functions that in grpc_device are listed as codegen no such as `CalFetchDate` so since I put them as codegen no in hapigen as well they no longer generate therefore these enums now cause a build error since they aren't referenced so they have been removed.